cmd/kube-apiserver/app/options/options.go

```diff
@@ -164,6 +169,9 @@ func (s *ServerRunOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.IntVar(&s.MasterCount, "apiserver-count", s.MasterCount,
 		"The number of apiservers running in the cluster, must be a positive number.")
 
+	fs.StringVar(&s.EndpointReconcilerType, "alpha-endpoint-reconciler-type", string(s.EndpointReconcilerType),
```

Comment:
I think it would be slightly more elegant to implement `pflag.Value` on `reconcilers.Type` and use `fs.Var`.

`s.ServiceNodePortRange` already does this as a precedent.

Reply:
that seems like overkill... there's more of a case to be made for a port range flag type than a single-use typedef

Reply:
Yeah, you're probably right.

I'm fine leaving it as is; I acknowledge this has hit bikeshedding levels.